climbtrackclient/src/components/organisms/LogonForm/LogonForm.js

```diff
@@ -1,16 +1,15 @@
-import React, { useState, useEffect } from 'react';
+import Button from '@material-ui/core/Button';
 import Paper from '@material-ui/core/Paper';
 import TextField from '@material-ui/core/TextField';
 import Typography from '@material-ui/core/Typography';
-import Button from '@material-ui/core/Button';
-import axios from 'axios';
-import { useDispatch } from 'react-redux';
-import { useSetAuthenticated } from '../../../dataLayer/actions/authenticationActions';
-import { useHistory, useLocation } from 'react-router-dom';
-import { useSelector } from 'react-redux';
-import { accessTokensExist, setAccessTokens } from '../../../dataLayer/accessToken/accessTokenHelper';
+import React, { useEffect, useState, useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { accessTokensExist } from '../../../dataLayer/accessToken/accessTokenHelper';
+import { login, useSetAuthenticated } from '../../../dataLayer/actions/authenticationActions';
+import { ErrorModal } from '../../atoms/ErrorModal/ErrorModal';
 
-const LogonForm = () => {
+export function LogonForm() {
```

Comment:
Generally why are functions rather than arrow functions considered to be more readable?

Reply:
I'm liking the named export thing...will move to it incrementally